test/grinder_files_test.dart

```diff
@@ -159,4 +159,81 @@ main() {
       expect(expectedResult, 'abcdefgh1234');
     });
   });
+
+  group('grinder.files Path', () {
+    Path temp;
+
+    setUp(() {
+      temp = Path.createSystemTemp();
+    });
+
+    tearDown(() => temp.delete());
+
+    test('Path.entity', () {
```

Comment:
Oh, I see it's for testing passing an entity to the constructor.  Maybe 'Path.entity' => 'Path(entity)'